### ISSUE TITLE
negative xrun times

### ISSUE THREAD

**Issue by swapan-g** [spatialaudio/jackclient-python#105]
Hello I am running the chatty client demo. the Xrun call back is producing negative values 
e.g.          xrun; delay -274886000640.0 microseconds

This clearly makes no sense.... There is no audible artifacts  in the audio stream

**Comment by mgeier** [comment]
Yeah, not only is it negative, it's also about 76 hours, which doesn't seem like a typical delay value ...

Maybe that's some kind of integer overflow error?

I don't think this is caused by the Python wrapper, the value comes directly from the JACK library (using the `jack_get_xrun_delayed_usecs()` function).

You should probably ask this to the JACK developers: https://jackaudio.org/community.html?

**Comment by swapan-g** [comment]
Thanks I will inquire there. For what it's worth: I also noticed values are the same magnitude but opposite in sign as the 'usecs' output when jack_showtime.  Also the values produces by the Xrun callback continually increase..

**Comment by mgeier** [comment]
@swapan-g Any news on this?
